src/test/suite/recorded.test.ts

```diff
@@ -111,26 +111,40 @@ async function runTest(file: string) {
   // Assert that recorded decorations are present
   checkMarks(fixture.initialState.marks, readableHatMap);
 
-  if (fixture.thrownError != null) {
-    await assert.rejects(
-      async () => {
-        await vscode.commands.executeCommand(
-          "cursorless.command",
-          fixture.command
```

Comment:
Note that the fact we had two callsites for executing the cursorless command resulted in this one rotting: we override `usePrePhraseSnapshot` below but not here 😳